gef.py

```diff
@@ -3584,6 +3588,21 @@ def safe_parse_and_eval(value):
     return None
 
 
+def parse_location(value):
+    """GEF wrapper for gdb.parse_and_eval() spezialied to resolve LOCATIONs like
+    '$pc' or 'main+5' """
+    try:
+        location = gdb.parse_and_eval(str(value))
+        ltype = location.dynamic_type.code
+        if ltype == 1 or ltype == 8:
```

Comment:
Maybe a comment of what these ltypes mean.

Reply:
Good call.

I will replace them with their names (https://sourceware.org/gdb/current/onlinedocs/gdb/Types-In-Python.html#Types-In-Python). Also obviously the checked types are not complete yet.

Currently this method also allows resolving symbols like function names. Therefore something like `cs main` works exactly as `cs &main`. This is because GDBs dynamic type code is set to `gdb.TYPE_CODE_FUNC`. 

This will not happen if `main` were a pointer instead of a function. Then the dynamic type code would be different and thereby not resolved the same way.